.travis.yml

```diff
@@ -16,8 +16,15 @@ jobs:
   include:
     - stage: Tests
       name: Unit Tests
-      script: mvn clean test --batch-mode
-      after_success: mvn -DrepoToken=$COVERALLS_REPO_TOKEN org.jacoco:jacoco-maven-plugin:report org.eluder.coveralls:coveralls-maven-plugin:report
+      before_script:
+        - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
```

Comment:
[minor] `curl --output cc-test-reporter` would be a more curl like way compared to redirecting the stdout to a file.